### PR TITLE
use correct ceph host dir for libvirt and nova

### PR DESCRIPTION
**Pull request by SeanMooney**
This change modifies the ceph dir that is mounted
to the libvirt qemu and nova comptute contaienr
from /etc/ceph  to /var/lib/openstack/config/ceph

(cherry picked from commit 81f2fe811aa8aac5a317ceb300a9c8ab37c05c99)

### CODE REVIEW

File: roles/edpm_nova/molecule/default/prepare.yml
Comment (rebtoor) [inline roles/edpm_nova/molecule/default/prepare.yml:83]: quote the octal number[1]

[1] https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-mode

Comment (SeanMooney) [reply roles/edpm_nova/molecule/default/prepare.yml:83]: this is a cherry-pick of an already merged patch we should not modify it

Comment (rebtoor) [reply roles/edpm_nova/molecule/default/prepare.yml:83]: Sorry Sean, i missed that it wasn't for `main` branch.

**Comment by rebtoor** [comment]
recheck

**Review by rebtoor — APPROVED** [review]

**Review by fmount — APPROVED** [review]
/lgtm

**Comment by SeanMooney** [comment]
so the content provider job failed but there are no logs

https://review.rdoproject.org/zuul/build/05c0ba84a25448fdb6f3e47418aff350

so we should perhaps just override this and merge.

**Comment by rebtoor** [comment]
> so the content provider job failed but there are no logs
> 
> https://review.rdoproject.org/zuul/build/05c0ba84a25448fdb6f3e47418aff350
> 
> so we should perhaps just override this and merge.

It will probably solved by this: https://github.com/openstack-k8s-operators/ci-framework/pull/681

Let's wait until it get merged.

**Comment by rebtoor** [comment]
recheck

**Comment by rebtoor** [comment]
recheck

**Pull request merged**
